S3/English/ASS_27.ass

```diff
@@ -0,0 +1,97 @@
+﻿[Script Info]
+; Script generated by Aegisub 3.2.2
+; http://www.aegisub.org/
+ScriptType: v4.00+
+WrapStyle: 0
+ScaledBorderAndShadow: yes
+YCbCr Matrix: TV.601
+PlayResX: 1920
+PlayResY: 1080
+
+[Aegisub Project Garbage]
+Audio File: 万圣街_1920x1080_万圣街_27.mp4
+Video File: 万圣街_1920x1080_万圣街_27.mp4
+Video AR Mode: 4
+Video AR Value: 1.777778
+Video Zoom Percent: 0.375000
+Scroll Position: 51
+Active Line: 40
+Video Position: 4066
+
+[V4+ Styles]
+Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
+Style: Default,Arial,45,&H00FFFFFF,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,18,0,20,1
+Style: Title,Arial,75,&H006CC2E9,&H000000FF,&H00000000,&H00000000,0,0,0,0,100,100,0,0,1,2,2,2,10,10,280,1
+
+[Events]
+Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
+Dialogue: 0,0:01:02.51,0:01:05.56,Title,,0,0,0,, {\pos(970,920)} {\b1\an5\pos(970,880)\fs120\frz4.5\t(0,100,1.5\fs45)}Episode 27:  {\c&H7990E7&}Panic as the house get crowded{\b0}
+Dialogue: 0,0:01:06.75,0:01:08.35,Default,,0,0,0,,This is a scene from "{\i1}A song of blood and brain{\i0}"
+Dialogue: 0,0:01:08.60,0:01:09.92,Default,,0,0,0,,A play with the vampire queen
+Dialogue: 0,0:01:10.82,0:01:11.82,Default,,0,0,0,,This is "{\i1}The last days{\i0}"
+Dialogue: 0,0:01:12.17,0:01:13.97,Default,,0,0,0,,It's a climax part I acted
+Dialogue: 0,0:01:14.06,0:01:14.84,Default,,0,0,0,,It's so amazing
+Dialogue: 0,0:01:16.88,0:01:19.72,Default,,0,0,0,,This video is the part I had a line in "{\i1}The bad omen{\i0}"
+Dialogue: 0,0:01:25.26,0:01:27.06,Default,,0,0,0,,The work is over for now
+Dialogue: 0,0:01:27.25,0:01:28.68,Default,,0,0,0,,Finally I can live here
+Dialogue: 0,0:01:28.77,0:01:29.50,Default,,0,0,0,,About this
+Dialogue: 0,0:01:29.81,0:01:30.24,Default,,0,0,0,,Luis
+Dialogue: 0,0:01:31.28,0:01:33.05,Default,,0,0,0,,1031 has no empty room left
+Dialogue: 0,0:01:35.07,0:01:36.54,Default,,0,0,0,,The room I used to live
+Dialogue: 0,0:01:36.54,0:01:37.58,Default,,0,0,0,,Now Neil lives there
+Dialogue: 0,0:01:39.20,0:01:40.70,Default,,0,0,0,,I remembered there was a room beside it
+Dialogue: 0,0:01:40.70,0:01:41.84,Default,,0,0,0,,Damao lives there
+Dialogue: 0,0:01:45.71,0:01:47.45,Default,,0,0,0,,That's fine
+Dialogue: 0,0:01:48.41,0:01:50.08,Default,,0,0,0,,How?
+Dialogue: 0,0:01:51.21,0:01:51.92,Default,,0,0,0,,Lynn
+Dialogue: 0,0:01:52.09,0:01:54.25,Default,,0,0,0,,How about let Luis sleep on the sofa like 
```

Comment:
"How about we letting Luis" or "How about we let Luis"
Also there's a "like" at the end, is there a part of the dialogue missing?